x-pack/plugins/security/server/authorization/authorization_service.tsx

```diff
@@ -64,8 +64,11 @@ interface AuthorizationServiceSetupParams {
   loggers: LoggerFactory;
   features: FeaturesPluginSetup;
   kibanaIndexName: string;
+
   getSpacesService(): SpacesService | undefined;
+
   getCurrentUser(request: KibanaRequest): AuthenticatedUser | null;
+
   customBranding: CustomBrandingSetup;
```

Comment:
__question:__ no preference, just curious since it looks unrelated to the main changes: are these changes done via some sort of automatic `eslint --fix` or ...?

Reply:
I have my IDEA configured to run prettier on save. It often lead to minor reformats such as adding white spaces between interface function defs. 

I can revert those changes if you wish.

Reply:
>I can revert those changes if you wish.

Nah, that's fine, I was just curious.